CONTRIBUTING.md

```diff
@@ -142,16 +145,7 @@ This processes is documented in [Letterbook.Adapter.Db](Letterbook.Adapter.Db/re
 
 ### Peers
 
-In addition to Letterbook's own dependencies, you may find it useful to have a 3rd party system to exercise federation. The `docker-compose.mastodon.yml` files provides a pre-configured Mastodon instance for that purpose. This is adapted with minimal tweaks from the Bitnami Mastodon config.
-
-```shell
-docker-compose -f docker-compose.mastodon.yml up
-```
-
-To log in:
-
-- email: `user@bitnami.org`
-- password: `bitnami1`
+In addition to Letterbook's own dependencies, you may find it useful to have a 3rd party system to exercise federation. The [Letterbook Sandcastles project][sandcastles] provides pre-configured instances of some other fediverse software. We encourage you to make use of that, and to contribute configurations for any other federated peers you're familiar with.
```

Comment:
Today I learned you could like other projects like that. Neat!

Reply:
That syntax is a markdown thing. You can put named links in the footer and then reference them like `[link text][link name]`, which I think is more readable for the times you have to read un-rendered markdown.